tests/test_record_medal.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from router.record import verify_medal
+from models import VerifyBody
+
+
+class TestRecordMedal(unittest.TestCase):
+    def setUp(self):
+        self.default_verify = VerifyBody(sport_name="Archery", participants=[
```

Comment:
you dont need to use VerifyBody inside it, just use a default python dictionary
eg.
```py
{
"sport_name": "Archery",
"participants": [{"country": "Thaiand", "medal": {"gold": 0, .....}}, ....]
```